Address Orie Steele's comment (#22)
lamps-wg/x509-shbs: draft-ietf-lamps-x509-shbs.md

- Software signing (Table IV of [CNSA2.0], [ANSSI])
- Certification Authority (CA) certificates.

In each of these cases the operator tightly controls their signing environment

> and is able to prevent OTS key reuse by employing state management strategies
> such as those listed in {{sec-security}} and backup and restore strategies
> ({{backup-restore}}).

Reviewer: ```suggestion
and is able to mitigate OTS key reuse by employing state management strategies from {{sec-security}} and backup and restore strategies from
({{backup-restore}}).
```
I think the strategies are rather mitigating reuse.
